docs/releasehistory.rst

```diff
@@ -15,6 +15,13 @@ Bugfixes
 - Bug in equilibration detection #1: The user was allowed to specify ``statistical_inefficiency`` without specifying ``n_equilibration_iterations``, which doesn't make sense, as ``n_equilibration_iterations`` and ``n_effective_max`` cannot be computed from ``statistical_inefficiency`` alone. Fixed by preventing user from specifying ``statistical_inefficiency`` without ``n_equilibration_iterations``. Issue `#609 <https://github.com/choderalab/openmmtools/issues/609>`_ (PR `#610 <https://github.com/choderalab/openmmtools/pull/610>`_). 
```

Comment:
We just discussed this, but just to set out the reminder: We want to start each of these with a clear description of the impact of the bug so users can tell if this would have impacted their work and may improve their simulations if they re-run them.

Reply:
@jchodera : Do you have suggestions on how to make this one clearer? I thought I had already made this one clear but maybe its still not clear enough

Reply:
I think that one is clear. It was probably the others that were unclear. I just committed the changes to these ones.